app/src/main/java/com/toygar/tasks/util/StringUtil.kt

```diff
@@ -0,0 +1,3 @@
+package com.toygar.tasks.util
+
+fun String.capitalizeFully() = split(" ").joinToString(" ") { it.capitalize() }
```

Comment:
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/to-title-case.html

Reply:
Char değil String olduğu için bu methodu kullanamıyorum.